src/components/combobox/combobox.tsx

```diff
@@ -688,11 +707,20 @@ export class Combobox
   };
 
   clickHandler = (event: MouseEvent): void => {
-    if (event.composedPath().some((node: HTMLElement) => node.tagName === "CALCITE-CHIP")) {
+    const composedPath = event.composedPath();
+
+    if (composedPath.some((node: HTMLElement) => node.tagName === "CALCITE-CHIP")) {
       this.open = false;
-      event.stopPropagation();
+      event.preventDefault();
```

Comment:
@macandcheese Tests seem to pass, but do you recall why the original event needed to stop propagating?

Reply:
It allowed focus to be set on the clicked Chip's close button, when user clicked the Chip, which was a change to support how Chips behaved post some Chip Group related changes.  

Although looking at it in action (and compared to past behavior), that probably wasn't needed, so this should be fine to remove.